app/services/doc_auth/errors.rb

```diff
@@ -120,7 +120,7 @@ module Errors
       GENERAL_ERROR => { long_msg: GENERAL_ERROR, field_msg: FALLBACK_FIELD_LEVEL, hints: true },
       # TODO, theses messages need modifying
       # Liveness, use general error for now
-      SELFIE_FAILURE => { long_msg: GENERAL_ERROR, field_msg: FALLBACK_FIELD_LEVEL, hints: false },
+      SELFIE_FAILURE => { long_msg: SELFIE_FAILURE, field_msg: SELFIE_NOT_LIVE_POOR_QUALITY_FIELD, hints: false },
```

Comment:
I'm wondering if the name of this constant also should be more specific, to match the `field_msg`. I think it might be confusing vs `SELFIE_NOT_LIVE` and `SELFIE_POOR_QUALITY`.

Unless this is supposed to be the general error. But then the field_msg seems to point at something more specific.

I'm not sure which way it should go, but the naming here at least seems like it's a little mismatched.

Reply:
As it turns out the problem was bigger. [Commit](https://github.com/18F/identity-idp/pull/10042/commits/e0afab4ddd1b50b0cd7a4854666024e54ec514c8). We already had this text (added since I started work on the PR). So I cleaned up the naming and the duplication. Nice catch!

Reply:
Thanks for fixing it!